users.go

```diff
@@ -945,7 +945,7 @@ type UserMembership struct {
 	SourceID    int    `json:"source_id"`
 	SourceName  string `json:"source_name"`
 	SourceType  string `json:"source_type"`
-	AccessLevel string `json:"access_level"`
+	AccessLevel int    `json:"access_level"`
```

Comment:
The docs say this is a string, are you sure this is an integer? If it really is an integer, we should probably use the [`AccessLevelValue`](https://github.com/xanzy/go-gitlab/blob/master/types.go#L38) type here?

Reply:
you can check this type, when i use AccessLevel with string the response is 400 by 'can not unmarshal num to string'

when i query gitlab code, the struct is define with number type
[project_member.rb](https://gitlab.com/gitlab-org/gitlab-foss/-/blob/master/qa/qa/resource/project_member.rb)

Reply:
Check... I'm cool with it, but the docs also give an example response that clearly shows a number returned as a string:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/4171547/103874911-0ad4bb00-50d2-11eb-80e8-2721dcd72354.png">

I'm not using GitLab myself that much lately, but I trust your test results. Which version of GitLab are you using? And in that case I guess we should use the `AccountLevelValue` as the type like I suggested. Agree?

Reply:
The version of GitLab that show v13.7.0. I agree with your suggestion， thanks!